### PR TITLE
Covery fixes in TBufferMerger test and add missing include in TMVA

### PR DESCRIPTION
**Pull request by amadio**

### CODE REVIEW

**Comment by amadio** [comment]
@phsft-bot build just on windows10/vc15

**Comment by amadio** [comment]
Rebuilding just on Windows. The `errx()` function used before is not available, replaced it with `perror()`.

File: io/io/test/TBufferMerger.cxx [hunk 1]
```diff
@@ -38,11 +38,19 @@ static bool FileExists(const char *name)
    return stat(name, &buffer) == 0;
 }
 
+static void RemoveFile(const char *name)
+{
+   if (remove(name) != 0) {
+      perror("failed to remove file");
+      exit(1);
```
Comment (pcanal) [inline io/io/test/TBufferMerger.cxx]: Consider gROOT->Terminate(1)

Comment (amadio) [reply io/io/test/TBufferMerger.cxx]: What is the difference?

**Pull request merged**